fatoptimizer/builtins.py

```diff
@@ -23,25 +23,24 @@ def _divmod_check_args(args):
 
 
 def _ord_check_args(args):
-    string = args[0]
-    return (len(string) == 1)
+    # param args[0] is a string
+    return (len(args[0]) == 1)
 
 
 def _bytes_check_args(args):
-    arg = args[0]
-    if not isinstance(arg, tuple):
+    if not isinstance(args[0], tuple):
         return True
     return all(0 <= item <= 255 for item in arg)
 
 
 def _pow_check_args(config, args):
-    num = args[0]
-    exp = args[1]
+    # num = args[0]
+    # exp = args[1]
     if len(args) >= 3:
-        mod = args[2]
+        return check_pow(config, args[0], args[1], args[2])
     else:
-        mod = None
-    return check_pow(config, num, exp, mod)
+        return check_pow(config, args[0], args[1], None)
```

Comment:
These two lines look duplicated. The purpose of the mod variable is to avoid code duplication which is a bad programming practice. Moreover, the new core looks less readable, it's hard to to guess the purpose of each parameter of the check_pow() call.